server/src/main/java/io/crate/session/Session.java

```diff
@@ -799,12 +799,12 @@ private static void emitRowCountsToResultReceivers(UUID jobId,
             try {
                 cells[0] = bulkResponse.rowCount(i);
                 cells[1] = bulkResponse.failure(i);
+                resultReceiver.setNextRow(row);
```

Comment:
I was about to add a change entry that CBE on ResultSet construction can leave stuck job entry - but that's true for 5.10.7, in 5.10.8 we wrongly kept processing due to regression and this is mentioned in the second commit.